ocs_ci/deployment/vmware.py

```diff
@@ -371,6 +371,18 @@ def __init__(self):
                 'installer'
             )
             self.previous_dir = os.getcwd()
+
+            # Download terraform binary based on ocp version and
+            # update in ENV_DATA
+            # use "0.11.14" for releases below OCP 4.5
+            ocp_version = get_ocp_version()
+            terraform_version = constants.TERRAFORM_OLD_VERSION
+            if Version.coerce(ocp_version) >= Version.coerce('4.5'):
+                terraform_version = config.DEPLOYMENT['terraform_version']
+            terraform_installer = get_terraform(version=terraform_version)
```

Comment:
Can you add a platform check for linux host and skip for others(mac) ? Thanks

Reply:
For mac, Do you want to skip or download appropriate one ?

Reply:
we should skip here, since its provided by brew.

Reply:
Vijay, thanks for pointing out, but the binary is working fine, we can use the binary for mac too.

Reply:
downloading the terraform according to platform in https://github.com/red-hat-storage/ocs-ci/pull/2394/files#diff-68bc5ed905b5a4cefbac287573f4f60fR2297